includes/Core/Remote_Features/Remote_Features.php

```diff
@@ -42,7 +42,19 @@ protected function get_type() {
 	 * @return array
 	 */
 	protected function get_default() {
-		return array();
+		return array(
+			'last_updated_at' => 0,
+		);
+	}
+
+	/**
+	 * Includes the current timestamp to the setting and updates it.
+	 *
+	 * @param array $features features array.
+	 */
+	public function update( $features ) {
+		$features['last_updated_at'] = time();
+		$this->set( $features );
```

Comment:
+ doc, but let's return the result.
```suggestion
		return $this->set( $features );
```